index.js

```diff
@@ -25,3 +25,4 @@ export { default as TV, initTV } from './src/TV'
 export { default as Pin, initPin } from './src/Pin'
 export { default as VideoPlayer, initVideoPlayer, mediaUrl } from './src/VideoPlayer'
 export { initLightningSdkPlugin } from './src/LightningSdkPlugins'
+export { default as SubtitlesParser } from './src/SubtitlesParser'
```

Comment:
I'm not sure if it makes sense for the users to actually have this exposed as a separate module, as it will always be used in conjunction with the `VideoPlayer` right? So it might as well be an internally used module. Or do you think otherwise? @Thomvl what are your thoughts on this?

Reply:
The idea is that it should be used along with other players like HLS or other native players, that's the reason we have exported this module.